src/christinaGerdzhikova/week01/DivideWithoutOP.java

```diff
@@ -20,5 +20,5 @@ public static void main(String[] args) {
         System.out.println(Divide(scan.nextInt(), scan.nextInt()));
```

Comment:
Make sure to check for division by zero

Reply:
I made it throw an exception if the denominator is zero